tests/test_generator_declarative.py

```diff
@@ -1592,3 +1594,81 @@ class WithItems(Base):
     str_matrix: Mapped[Optional[list[list[str]]]] = mapped_column(ARRAY(VARCHAR(), dimensions=2))
 """,
     )
+
+
+@pytest.mark.parametrize("engine", ["postgresql"], indirect=["engine"])
+def test_domain_json(generator: CodeGenerator) -> None:
```

Comment:
Perhaps this should be parametrized to check for both JSON and JSONB? Assuming that it's a simple enough replacement.

Reply:
Ok, will do so

Reply:
Fixed